### ISSUE TITLE
[docs] Grid lines for activation function plots

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#9130]
Without them it's hard to understand the right asymptote of sigmoid, and the linear regime bounds (more of a pedagogical thing for new users)

https://pytorch.org/docs/stable/nn.html?highlight=multilabel_soft_margin_loss#torch.nn.Sigmoid

Even very light lightgray dotted grid lines would be nice

**Comment by vadimkantorov** [comment]
Should we also add 1/-1 grid lines? For many activation functions the asymptotes are at 1/-1, not at 2/-2
